readme.md

```diff
@@ -0,0 +1,178 @@
+<a name="readme-top"></a>
+
+<div align="center">
+   <br/>
+
+  <h3><b>Expense App</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 About the Project ](#-about-the-project-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ ](#-faq-)
+  - [📝 License ](#-license-)
+
+# 📖 About the Project <a name="about-project"></a>
+
+**Expense App** is an transaction tracker app containing data about transactions based on categories. A user can register/login and create transactions to track them.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Databases</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Contains data about categories**
+- **Contains data about transactions**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Link to Live demo](https://www.loom.com/share/b6bc3659c096497d8033110ade38c3f6?sid=544571df-58df-4030-a0f0-b49960266c29)
```

Comment:
- Impressive project presentation, @obibaadoma! 👍🏽
I noticed there are two `readme.md` files. The primary one on the setup branch lacks the recent update where you included the Loom link for the presentation video. Additionally, the Live Demo link on the setup branch remains empty. Could you please make the necessary adjustments to ensure consistency and completeness in the readme.md on the setup branch? Your attention to this matter is appreciated. Thank you!

https://ibb.co/p2ZJr8J